docs/README.md

```diff
@@ -168,6 +168,45 @@ Called when a participant has disconnected
 
 @param {{roomName, participant}}
 
+#### setLocalVideoEnabled
+
+```js
+setLocalVideoEnabled: Function
+```
+
+Called when a local video is disable / enabled, how to use it below
+
+```js
+_onDisableCameraButtonPress = () => {
+  this.twilioVideo
```

Comment:
Elsewhere in the example code we use `this.refs.twilioVideo`.  Mind changing that here to be consistent?

Reply:
Sure this has been updated!